src/openrct2/config/Config.cpp

```diff
@@ -137,6 +137,7 @@ namespace Config
             auto model = &gConfigGeneral;
             model->always_show_gridlines = reader->GetBoolean("always_show_gridlines", false);
             model->autosave_frequency = reader->GetInt32("autosave", AUTOSAVE_EVERY_5MINUTES);
+            model->autosave_amount = reader->GetInt32("autosave_amount", 8);
```

Comment:
Where does this `8` come from? The current autosave limit is 10.

Reply:
Strangely enough, even though a new autosave was made, there are still 10 saved games in my autosave folder.